wave_vision/config/descriptor/brisk.yaml

```diff
@@ -0,0 +1,25 @@
+# Configuration parameters for BRISK Descriptor Extractor
+
+# radiusList defines the radius of each subsequent circle (in pixels). All
+# numbers must be positive. Cannot be empty.
+#
+# Default: radiusList: [0.0, 2.465, 4.165, 6.29, 9.18]
+radiusList: [0.0, 2.465, 4.165, 6.29, 9.18]
+
+# numberList defines the number of points in each circle. Must be the same
+# size as radiusList. All numbers must be positive. Cannot be empty.
+#
+# Default: numberList: [1, 10, 14, 15, 20]
```

Comment:
These default values are the ones you would get if you create the "other" BRISK constructor (the values  when you initialize the BRISK detector). These values are hardcoded into the other constructor.

Maybe default isn't the right word....recommended, perhaps?